docs/02-local-development.md

```
@@ -40,7 +40,7 @@ graph LR
 === "Python Runtime"
 
     ```bash
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)